IOSShared/Foundation/NSString+SDExtensions.m

```diff
@@ -105,14 +105,14 @@ - (NSString*)stripHTMLFromListItems {
 
 
 - (NSString*)escapedString 
-{            
-	NSString *selfCopy = [self mutableCopy];
-	return (__bridge_transfer  NSString *) CFURLCreateStringByAddingPercentEscapes(NULL, (__bridge CFStringRef)selfCopy, NULL, CFSTR("%￼=,!$&'()*+;@?\n\"<>#\t :/"), kCFStringEncodingUTF8);
+{
+    NSCharacterSet *allowed = [NSCharacterSet characterSetWithCharactersInString:@"-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz~"];
```

Comment:
@regnerjr I chose this character set to match the behavior of `CFURLCreateStringByAddingPercentEscapes(…"%￼=,!$&'()*+;@?\n\"<>#\t :/"…)`, which escapes everything except the characters in the chosen set. (I verified this by running it on a string containing all possible ASCII characters, as well as some 8-bit values.) In practice, it probably wouldn't make any difference, but this provides the most exact backwards compatibility.